content/docs/2.6/scalers/aws-dynamodb.md

```diff
@@ -0,0 +1,161 @@
++++
```

Comment:
We have already shipped v2.6 - Would you mind moving it to the v2.7 folder and update the availability here please?